packages/renderer/src/lib/ui/FileInput.svelte

```diff
@@ -11,10 +12,13 @@ export let options: OpenDialogOptions;
 export let readonly: boolean = false;
 export let required: boolean = false;
 
+const dispatch = createEventDispatcher<{ change: string }>();
```

Comment:
as event dispatcher are deprecated in svelte5, is it possible to use a callback method rather than introducing an event dispatcher ?

Reply:
https://svelte-5-preview.vercel.app/docs/event-handlers

Reply:
i didn't know... updated 👍